services/CollectionService/src/index.php

```diff
@@ -25,40 +22,25 @@
   'twig.path' => __DIR__.'/../templates',
 ));
 
-$app['twig'] = $app->share($app->extend('twig', function($twig, $app) {
-  return $twig;
-}));
 $islandoraResourceServiceProvider = new \Islandora\ResourceService\Provider\ResourceServiceProvider;
 
-$app->register($islandoraResourceServiceProvider);
+//Registers Resource Service and defines current app's path for config context
+$app->register($islandoraResourceServiceProvider, array(
+  'islandora.BasePath' => __DIR__,
+));
 $app->mount("/islandora", $islandoraResourceServiceProvider);
 $app->register(new \Islandora\ResourceService\Provider\UuidServiceProvider(), array(
   'UuidServiceProvider.default_namespace' => $app['config']['islandora']['defaultNamespaceDomainUuuidV5'],
 ));
 
-//$app['uuid'] = $app['islandora.uuid5'](rand());
+//We will use a static uuid5 to fake a real IRI to replace <> during turtle parsing
+$app['uuid5'] = $app->share(function () use ($app) {
```

Comment:
I understand that you're using this as a dummy value to keep EasyRDF happy.  I have no issue with that, nor how you're generating the value.  What I'm saying is that you're not using this function anywhere else except to generate the fake id, right?  So skip the middle man and just put the fake ID in there.

My second point is to avoid using the DI container as a cache for variables.  You'll quickly wind up with a universal resource locator, or god object, not a DI container.  You can just make a variable in the index.php scope, or even better, put it in the class we create when refactoring away from a fat controller.  This can wait for later, though.